### PR TITLE
feat: SQL Policy tests

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: repo-settings/providers/k8s/config.yaml [hunk 1]
```diff
@@ -14,4 +14,5 @@ branchProtectionRules:
     - Lint Provider Doc
     - test_unit (postgres:10, 1.17, ubuntu-latest)
     - release-dry-run
-    - PolicyTest (postgres:10, 1.17, ubuntu-latest)
\ No newline at end of file
+    - PolicyTest (postgres:10, 1.17, ubuntu-latest)
```
Comment (yevgenypats) [inline repo-settings/providers/k8s/config.yaml:17]: This is not needed any more

**Review by yevgenypats — DISMISSED** [review]
Looks good. one comment

File: repo-settings/providers/with-terraform-and-policy-tests/config.yaml [hunk 2]
```diff
@@ -16,4 +16,5 @@ branchProtectionRules:
     - release-dry-run
     - test_integration (postgres:10, 1.17, ubuntu-latest)
     - Terrafrom Plan
-    - PolicyTest (postgres:10, 1.17, ubuntu-latest)
\ No newline at end of file
+    - PolicyTest (postgres:10, 1.17, ubuntu-latest)
```
[the remark this answers is not in the document]
Comment (disq) [reply repo-settings/providers/with-terraform-and-policy-tests/config.yaml]: removed

File: workflows/providers/test_policy_sql.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    CGO_ENABLED: 0
+    CQ_NO_TELEMETRY: 1
+    PGPASSWORD: pass
+
+  jobs:
+    SQLPolicyTest:
+      strategy:
+        matrix:
+          dbversion: [ "postgres:10" ]
+          go: [ "1.17" ]
```
Comment (hermanschaaf) [inline workflows/providers/test_policy_sql.yml:24]:
```suggestion
          go: [ "1.18" ]
```

Comment (disq) [reply workflows/providers/test_policy_sql.yml]: oops, fixed

File: workflows/providers/test_policy_sql.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
+  {{=<% %>=}}
```
[the remark this answers is not in the document]
Comment (disq) [reply workflows/providers/test_policy_sql.yml:1]: not sure probably testing templating... it's there in all/most other yaml files so I left it in place here as well.

**Review by hermanschaaf — APPROVED** [review]

File: workflows/providers/test_policy_sql.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (erezrokah) [reply workflows/providers/test_policy_sql.yml:1]: The templating bit is documented in https://github.com/cloudquery/.github/tree/155afcb1f5b31a29a02729ac9e390e3460c35231#sync-files-templating

**Pull request merged**